src/main/java/com/rk/rentacarapp/services/VehicleService.java

```diff
@@ -0,0 +1,26 @@
+package com.rk.rentacarapp.services;
+
+import com.rk.rentacarapp.model.Vehicle;
+import com.rk.rentacarapp.model.VehicleType;
+
+import java.util.List;
+
+public interface VehicleService {
+
+    List<Vehicle> findAllVehicles();
```

Comment:
getAll()

vehicleService.getAll() with a return type of List<Vehicle> most likely gives me all vehicles